.github/workflows/docs-sphinx-python-dependency-build-checks.yml

```diff
@@ -11,9 +11,9 @@
 # succeed.
 name: Check and document build requirements for Sphinx venv
 on:
-  - push
-  - pull_request
-  - workflow_dispatch
+  schedule:
+    - cron: "0 2 * * 0,3" # Runs at 02:00 AM on every Sunday and Wednesday.
```

Comment:
Let's not run this on Sunday. Can we have it run on Monday and Friday please, so APAC is around to fix any issues.

Also keep workflow_dispatch so we can trigger this manually.

Reply:
Changed to Monday and Friday.

Makes sense, added workflow dispatch back in. Hadn't realised that it would disable that.